server/proto/attestation.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package attestation;
+
+message AttestationRequest {
+    bytes evidence = 1;
+    optional string user = 2;
```

Comment:
I would create a `User` gRPC type and use that instead of a raw user ID string. That will allow to isolate our `User` definition from the gRPC API itself.